docs/src/main/asciidoc/config-yaml.adoc

```diff
@@ -1,26 +1,23 @@
 ////
-This guide is maintained in the main Quarkus repository
-and pull requests should be submitted there:
+This guide is maintained in the main Quarkus repository, and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= YAML Configuration
+= YAML configuration
 include::_attributes.adoc[]
 :categories: core
 :summary: YAML as a Configuration Source.
 
-https://en.wikipedia.org/wiki/YAML[YAML] is a very popular format. Kubernetes relies heavily on the YAML format to
-write the various resource descriptors.
+link:https://en.wikipedia.org/wiki/YAML[YAML] is widely used, especially in Kubernetes, for defining resource descriptors.
+With {ProductLongName}, you can use YAML as an alternative to the standard Java Properties file for configuration.
```

Comment:
The {ProductLongName} attribute does not exist upstream.  Upstream docs arbitrarily use {project-name}.  Where:
```
:project-name: Quarkus
```
Suggest we either:
- Use {project-name} and we can set that in our local downstream attributes file to:

```
:project-name: Red Hat build of Quarkus
```

- Ask upstream community to adjust naming attribute and move to local attributes, whereby we can modify as we need. 

- Ask upstream to help us to develop a postscript as part of the doc build downstramdoc.java to find Quarkus or {project-name} and set it to {ProductLongName} or something else.

WDY? 

@rolfedh - I think this overlaps with some other good approaches too in your prototype suggestion...

Reply:
Updated. Thanks.